atlp-rwanda/team-emma-25-e-comm-backend
# ft-twoFactorAuthentication-#184359075

#### What does this PR do?
This ensures two-factor authentication for a seller.

#### Description of Task to be completed?

Users that require a higher level of security (Sellers) should be able to be verified where the application also includes an extra factor i.e SMS or Authentication codes.

#### How should this be manually tested?
This feature has 2 endpoints one for sending the AUTHCODES to the seller's phone number
`/sendcode/:phone`
`/verify/:phone/:code`
Use Postman client to test these routes
⚠on `:phone` write the phone with the country code. Example: +25078--------- 

#### What are the relevant pivotal tracker/Trello stories?
STORY_ID #184359075
